apps/omg_eth/test/omg_eth/root_chain_test.exs

```diff
@@ -29,31 +29,28 @@ defmodule OMG.Eth.RootChainTest do
     vcr_path = Path.join(__DIR__, "../fixtures/vcr_cassettes")
     ExVCR.Config.cassette_library_dir(vcr_path)
 
-    contract = %{
-      # NOTE: achiurizo
-      # this has changed with ALD(see plasma-contrats deploy of plasma_framework)
-      # it's now :plasma_framework_tx_hash instead of :txhash_contract
-      txhash_contract: Encoding.from_hex("0x3d517d431daea71a99310f12468ffdf2bf547ad1d148f42acfc4ee34dd4e84d7"),
-      plasma_framework: Encoding.from_hex("0xd17e1233a03affb9092d5109179b43d6a8828607"),
-      eth_vault: Encoding.from_hex("0x1967d06b1faba91eaadb1be33b277447ea24fa0e"),
-      erc20_vault: Encoding.from_hex("0xaef6182310e3d34b6ea138b60d36a245386f3201"),
-      payment_exit_game: Encoding.from_hex("0x902719f192aa5240632f704aa7a94bab61b86550"),
-      authority_address: Encoding.from_hex("0x22d491bde2303f2f43325b2108d26f1eaba1e32b")
+    contracts = %{
```

Comment:
these could also get read from file. like `deffixture contract(eth_node) do`. wdyt?

Reply:
Sounds good. Looks like `deffixture contract` is being used in only one test file I should be able to remove this fixture completely. Let me do it in a separate PR.